if(line_constructor_start != -1 && line_constructor_end != -1)
    {
        auto it = lines.begin() + line_constructor_end;

        for(int i = line_constructor_start; i < line_constructor_end; ++i)
            lines[i] = "\t" + lines[i];

        for(auto& call: calls)
            lines.insert(it, "\t" + call);

        it += calls.size();
        auto it_start = lines.begin() + line_constructor_start;

        lines.insert(it, std::string("        except Exception as ex:\n            print('Exception caught in file %s !' % __file__)\n            print(type(ex).__name__)\n            print(ex.args)"));
        lines.insert(it_start, std::string("        try:"));
    }

// Exception caught in file /home/robert/si/si0.12.1/plugins/__loaded_plugins__/standard_environment_library/crash_scenario/Crash.py !
// StopIteration